app/locales/en/translations.js

```diff
@@ -65,6 +65,7 @@ export default {
             },
             otherRepositories: 'Other registraation repositories',
         },
+        type_error: 'An error occurred getting registration types.  Please try again.',
```

Comment:
I think I'd prefer
`registration_metaschema_error` or something -- type error makes me think something completely different: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/TypeError

Reply:
Also, there is an extra space before `Please`